src/compiler/scala/tools/nsc/PhaseAssembly.scala

```diff
@@ -33,7 +33,7 @@ trait PhaseAssembly {
 
     /**
      * Simple node with name and object ref for the phase object,
-     * also sets of in and out going dependencies
+     * also sets of in- and out-going dependencies.
```

Comment:
incoming and outgoing? earning money and extroverts? Or maybe you pick up your co-dependent friend to grab a hamburger, in which case you would be "In'n'Out-going".

Reply:
```suggestion
     * also sets of incoming and outgoing dependencies.
```